### DISCUSSION TITLE
Welcome to SLM-Lab Discussions!

### DISCUSSION THREAD

**Discussion by kengz** [kengz/SLM-Lab#470] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cnatale** [comment]
Chris Natale, he/they. I work as a web software engineer and picked the related book up because I'm interested in deep reinforcement learning. I've done some machine learning programming in the past, mostly 5-8 years ago. It's amazing how much the field has advanced since then!

My hobby project rn is attempting to teach a robot locomotion using deep reinforcement learning.
